zap/src/parser/reports.rs

```diff
@@ -35,6 +35,10 @@ pub enum Report<'src> {
 
 	AnalyzeEmptyEvDecls,
 
+	AnalyzeMissingEvDeclType {
+		span: Span,
+	},
```

Comment:
nit: all instances of `AnalyzeMissingEvDeclType` should be in order of error codes like the rest of them.